config.json

```diff
@@ -604,6 +604,37 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Simple Syntax",
+      "content": "Crystal's syntax is heavily inspired by Ruby, so it is natural to read and easy to write.",
+      "icon": "TODO"
+    },
+    {
+      "title": "Static Type Checking",
+      "content": "Crystal is statically type checked with built-in type inference, so type errors are caught early by the compiler.",
```

Comment:
This exceeds the maximum of 100 characters.

Reply:
```suggestion
      "content": "Crystal is statically type checked with built-in type inference, so type errors are caught early.",
```